app/Http/Controllers/ImportController.php

```diff
@@ -70,6 +73,11 @@ public function store(Request $request): JsonResource
 
         $meta->save();
 
+        Bus::chain([
```

Comment:
What is a Bus?? O: the new things keep on coming!!

Reply:
A bus is essentially (I might be slightly inaccurate here) a mechanism that is responsible for the order in which jobs are put in a queue. In this case, what we are trying to do is to "bus together" the CSV file validation job and the CSV import job. But currently, since this is a WIP - it is unsuccessful, and I'm trying to understand why :disappointed:

Reply:
Argh, found it! I was using `->map()` instead of `->each()` :roll_eyes: